### ISSUE TITLE
Move placeholder content in SectionSidePanel to own component

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#11275]
# Overview

**NOTE** Check out my `feature--finalizing-create-quizz` (yes 2 `z`'s) and base work on this from there. Once that is merged in #11277, you can rebase against `develop` when you make your PR to avoid convoluted commit history issues

There is currently some content that would do well in its own little component to clean things up in the SectionSidePanel component.

When a section has no questions selected, we present them with a message and a button to go to select some resources. [FIGMA](https://www.figma.com/file/wRlClsHgcVqIwgUrntRN1a/Kolibri-Enhanced-Quiz-Management?type=design&node-id=28-7898&mode=design&t=zsfffy4qmp7fH8xj-0)

Where this is, we will soon be putting another component's worth of stuff, so to preemptively keep this clean, we should make a child component for the empty state and conditionally show it if there are questions available.

See `useQuizCreation` for the data module. You _might_ need to `inject: ['quizForge']` in the new component and you can then access any export defined in `useQuizCreation` from that object -- ie, `quizForge.activeQuestionsPool.value` will give you the list of questions available. Note that wherever you see `quizForge` injected, that is your entry into the same thing.

```javascript
  /** @type {ComputedRef<QuizQuestion[]>} All questions in the active section's `resource_pool`
   *                                      exercises */
  const activeQuestionsPool = computed(() => _getQuestionsFromSection(get(_activeSectionId)));

```

#### Acceptance Criteria

- [ ] A new component is created that houses "There are no questions in this section" UI
- [ ] The component will employ the `quizForge` object as needed to hide/show depending on if there are questions or not

**Comment by rtibbles** [comment]
This is no longer relevant.
